### ISSUE TITLE
Dynamic position range for keypad

### ISSUE THREAD

**Issue by DianaTavares** [phetsims/my-solar-system#134]
I know that position limits by using the number pad are tricky for the zooms. I just find it very restrictive with the zoom and the control just allows them a couple of AUs. For example, in the screenshot, the pad only allows -2 to 2, and the play area is 17 AU. 

![image](https://user-images.githubusercontent.com/26031501/227385406-5d4c6d85-ad77-40c7-a802-f13f6a636936.png)

Can the limits of the position being dynamic according to the zoom? 
Same for x axis.

**Comment by AgustinVallejo** [comment]
Changing the position limits in the keyboard. And also marking as deferred since NumberKeyPad does not support a dynamic range. But could be interesting to implement for a further release.

**Comment by pixelzoom** [comment]
It sounds like you were using NumberKeydad at some point, which is deprecated.  But now I see that you're using Keypad, via KeypadDialog (see InteractiveNumberDisplay.ts).   And dynamic range is very much supported -- just pass the desired range to `keypadDialog.beginEdit`.

The bigger issue is that NumberDisplay (the superclass of InteractiveNumberDisplay) does _not_ support a dynamic range. But I believe that its `displayRange: Range` parameter is only used to size the NumberDisplay.  So you could just use the largest range that you will need, and it should work fine.

**Comment by pixelzoom** [comment]
I'm also guessing that this is not specific to Position (x, y).  The range for Velocity (Vx, Vy) could also be adjusted by zooming.

**Comment by pixelzoom** [comment]
I'm going to remove the deferred label from this, because it seems like we should evaluate this for the 1.3 PhET-iO release.

**Comment by pixelzoom** [comment]
@arouinfar Let's discuss (1) whether to address this for the 1.3 release, and if so (2) what the dynamic range of position should be for each zoom level.

**Comment by pixelzoom** [comment]
I played around with this, and discovered that the position range is currently x=[-8,8] and y=[-4,4] for all zoom levels.  That seems to be tuned to zoomLevel=2, which looks like this:

<img width="1034" alt="screenshot_2891" src="https://github.com/phetsims/my-solar-system/assets/3046552/4bf26a50-7907-4b73-8761-d32a2632ef99">

For zoomLevel=1 (full zoomed out), those ranges are limiting:

<img width="1037" alt="screenshot_2892" src="https://github.com/phetsims/my-solar-system/assets/3046552/bac15008-d69a-4f6b-ac3e-833d7f1a7299">

Dynamic zoom ranges feels like a can of worms, and I don't think we should go there.  But I do think we might consider setting the ranges to correspond to zoomLevel=1.    That would be x=[-14,14] and y=[-8,8].

@arouinfar @DianaTavares @AgustinVallejo thoughts?

**Comment by pixelzoom** [comment]
@arouinfar @AgustinVallejo and I met to discuss.

We decided that a dynamic range that is a function of zoom level could be confusing, especially in a classroom situation where the teacher is instructing students to set position values via the keypad.  And it's also unnecessarily limiting, because it requires you to zoom before setting values.

We did decide that expanding the range to x=[-14,14] and y=[-8,8] is beneficial, and that was done in https://github.com/phetsims/my-solar-system/commit/172700b36654a87c54c76c5c1350fc83a1ede7b9.    Since I tested and demo'ed this during our meeting, I'll go ahead and close this issue.
